documentation/src/main/asciidoc/topics/proc_enabling_jmx.adoc

```diff
@@ -3,6 +3,8 @@
 You can enable JMX with {brandname} servers to collect statistics and perform
 administrative operations.
 
+Start the server with the addictional JVM arguments: `-Dcom.sun.management.jmxremote.port=9999 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false`
+
```

Comment:
`s/addictional/additional`, but I think the following might be better:

"Start the server with the following properties: "

Reply:
Done

Reply:
@oraNod what should be the file that I need to include this part?

Reply:
@diegolovison hmm. I think those system properties still count as 3rd party and we might not want to show users how to disable auth/encryption. 

Perhaps we could add a note towards the bottom saying that you can pass `com.sun.management.jmxremote.*` properties to configure JMX client connections. We could also reference Java docu like this: https://docs.oracle.com/javase/8/docs/technotes/guides/management/agent.html 

For the server, it might be worth adding a "cheat sheet" for system properties to https://github.com/infinispan/infinispan/blob/master/server/runtime/src/main/server/README.md 

or https://github.com/infinispan/infinispan/blob/master/server/runtime/README.adoc 

A wild idea might be to add JMX config options to the CLI so you can avoid system props with the server altogether.

Reply:
I thnk we can close the PR. Thanks

Reply:
thanks for submitting a doc PR though @diegolovison ! it's always good to try and find ways to improve content and share knowledge.